bindings/wrap/tests/types/polywrap.app.yaml

```
@@ -0,0 +1,6 @@
+format: 0.3.0
+project:
+  name: template-wasm-rs-app-types
```

this can be renamed to something like
```suggestion
  name: tiktoken-types
```